Gaijinizer patch/CommonEvents_95.txt

```diff
@@ -2,7 +2,7 @@
 > BEGIN STRING
 ヤッホー、元気ぃ？
 > CONTEXT: CommonEvents_95/0/Dialogue/enemy
-hey!#MTLed
+Hi hi! How ya going?#Ron
```

Comment:
I'm going to go out on a limb here and say you meant to say "doing" instead of "going"?

Reply:
This might be a cultural thing. In my part of the world (Australia/NZ) "How are you going?" is a common greeting.